packages/siop-oid4vp/lib/authorization-response/OpenID4VP.ts

```diff
@@ -279,7 +281,7 @@ export const assertValidVerifiablePresentations = async (args: {
     hasher?: Hasher
   }
 }) => {
-  const presentationsArray = Array.isArray(args.presentations) ? args.presentations : [args.presentations]
+  const presentationsArray = args.presentations ? (Array.isArray(args.presentations) ? args.presentations : [args.presentations]) : []
```

Comment:
Presentations should always be present. See remark above. If you want to make a change then it should be an assertion we actually have at least one Presentation and fix the calling code. No Presentation doesn't exist with oid4vp. There is always at least one

Reply:
fixed